doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -0,0 +1,257 @@
+.. _ug_nrf54h20_architecture_pm:
+
+nRF54H20 Power Management
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54H20 SoC is a distributed system where each domain tries to achieve minimal power consumption for itself.
+When all CPUs are ready to fully minimize power consumption by entering the System Off hardware state, the System Controller prepares the system and triggers the entrance in this state.
+
+Power states
+************
+
+The nRF54H20 ARM Cortex CPUs currently support the following software power states:
+
+* Active
+* Idle
+* Local suspend to Ram
+
+In following releases on the |NCS|, support for the following power states will be added:
+
+* Idle with cache retained in RAM
+* Idle with cache disabled
+* Suspend to RAM ready for System Off
+* Soft Off
+
+Overview
+********
+
+Each CPU in the nRF54H20 SoC tries to preserve as much power as possible, independently from other CPUs.
+The power management subsystem, operating independently within each CPU, continuously selects the most optimal power state based on the current conditions of the CPU.
+
+Power states details
+********************
+
+The following sections describes the details of each of the software power states available on the nRF54H20 SoC.
+
+Active
+======
+
+*Active* is the power state in which the CPU is actively processing.
```

Comment:
Should the power states be made italic. I haven't see these mentioned as Italic in other docs.

Reply:
The problem I see is with power states having a long name like "Local suspend to Ram". Not highlighting it in some way might render sentences containing it quite difficult to read. Not necessarily they have to be in Italics, but I believe that they should be highlighted.